test-services/src/main/resources/log4j2.properties

```diff
@@ -9,10 +9,10 @@ appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} [%tn] %-5p %notEmpty{[
 
 # Restate logs to debug level
 logger.app.name = dev.restate
-logger.app.level = debug
+logger.app.level = ${env:RESTATE_LOG_LEVEL:-debug}
```

Comment:
In the typescript SDK we are using `RESTATE_LOGGING`. Does it make sense to unify them?